### ISSUE TITLE
Persona. Clickable and closeable components accessibility

### ISSUE THREAD

**Issue by Kyzyl-ool** [gravity-ui/uikit#966]
[Clickable](https://preview.gravity-ui.com/uikit/?path=/story/components-persona--clickable) component is not accessible from keyboard. In [Closeable](https://preview.gravity-ui.com/uikit/?path=/story/components-persona--closable) one the close button is not accessible
